README.md

```diff
@@ -31,4 +31,19 @@ Then import it into python.
 ## Testing
 
 This package uses [pytest](https://pytest.org/) for tests.
-To run tests locally, execute the command `pytest` from the command line.
+To run tests locally, execute the command `pytest` from the command line:
+
+```
+pytest
```

Comment:
Why don't we use

```
>>>pytest
```
seems to be a convention for CLI statements. We can do this everywhere too.. all of my install instructions should be updated following this!

see
https://github.com/mapbox/rasterio/blob/master/README.rst
for an example

Reply:
I think rasterio is using `$` to indicate bash commands, and `>>>` to indicate Python commands. So, to be consistent with them maybe I will change this to 

```bash
$ pytest
```

Reply:
Ok I have changed all bash commands to use $ and all python commands to use >>>

Reply:
woohoo!!